sky/unit/test/widget/build_utils.dart

```diff
@@ -114,4 +129,13 @@ class WidgetTester {
     Component.flushBuild();
     RenderObject.flushLayout();
   }
+
+  void pumpPaintFrame(WidgetBuilder builder) {
```

Comment:
Can we just make pumpFrame do this?  I'd like to avoid having a separate function.